worksheet_B/readMe.md

```diff
@@ -0,0 +1,23 @@
+
+START
+password = GENERATE(Four letter word)      //Creates a four letter word.
+attempts = 4 i = 4while(i>0)                             //Creates a while loop for each guess the user has.
```

Comment:
I think your formatting has become messed up somehow, which makes your pseudocode difficult to follow.